examples/prod/README.md

```diff
@@ -0,0 +1,135 @@
+│  │ │ (Dedicated) │ │  │ │ (Dedicated) │ │  │ │ (Dedicated) │ │ │
+│  │ └─────────────┘ │  │ └─────────────┘ │  │ └─────────────┘ │ │
+│  │                 │  │                 │  │                 │ │
+│  │ ┌─────────────┐ │  │ ┌─────────────┐ │  │ ┌─────────────┐ │ │
+│  │ │ Worker      │ │  │ │ Worker      │ │  │ │ Worker      │ │ │
+│  │ │ (Optimized) │ │  │ │ (Optimized) │ │  │ │ (Optimized) │ │ │
+│  │ └─────────────┘ │  │ └─────────────┘ │  │ └─────────────┘ │ │
+│  └─────────────────┘  └─────────────────┘  └─────────────────┘ │
+│                                                                 │
+│  ┌─────────────────┐  ┌─────────────────┐  ┌─────────────────┐ │
+│  │ Load Balancer   │  │ Monitoring      │  │ Backup &        │ │
+│  │ (ALB + NLB)     │  │ (CloudWatch)    │  │ Recovery        │ │
+│  └─────────────────┘  └─────────────────┘  └─────────────────┘ │
+└─────────────────────────────────────────────────────────────┘
+```
+
+## Requirements
+
+- Terraform (v1.5.7+)
+- AWS CLI with enterprise-level permissions
+- Compliance and security requirements documentation
+- Backup and disaster recovery procedures
+- Monitoring and alerting setup
+
+## Pre-Deployment Checklist
+
+- [ ] Review and approve security configurations
+- [ ] Validate compliance requirements
+- [ ] Set up monitoring and alerting
+- [ ] Configure backup and recovery procedures
+- [ ] Test disaster recovery procedures
+- [ ] Document operational procedures
+- [ ] Set up access controls and IAM policies
+
+## Usage
+
+1. **Security Review**: Review all security configurations in `main.tf`
+2. **Compliance Check**: Ensure configurations meet compliance requirements
+3. **Variable Configuration**: Set production-specific variables
+4. **Deploy**: `terraform plan` and `terraform apply`
+5. **Validation**: Run post-deployment tests and validations
+
+## Operational Considerations
+
+### Maintenance Windows
+- Schedule maintenance during low-traffic periods
+- Use rolling updates to minimize downtime
+- Test all changes in staging environment first
+
+### Monitoring & Alerting
+- Set up comprehensive monitoring dashboards
+- Configure alerts for critical metrics
+- Establish escalation procedures
+
+### Backup & Recovery
+- Test backup and recovery procedures regularly
+- Document recovery time objectives (RTO) and recovery point objectives (RPO)
+- Maintain disaster recovery runbooks
+
+### Security
+- Regular security audits and penetration testing
+- Update security patches and configurations
+- Monitor for security threats and vulnerabilities
+
+## Cost Optimization
+
+Production environments can be expensive. Consider:
+- **Reserved Instances**: For predictable workloads
+- **Spot Instances**: For non-critical workloads
+- **Storage Optimization**: Use appropriate storage classes
+- **Network Optimization**: Minimize cross-AZ traffic
+- **Resource Right-sizing**: Monitor and adjust resource allocation
+
+## Compliance & Governance
```

Comment:
This module doesn't have security concerns like this in mind.
RKE2 in general does, but this module's examples don't have any special considerations for compliance.